### PR TITLE
Feat/repo initialization

### PR DESCRIPTION
**Pull request by eloiGarrido**
Commit initializing the repository and containing required NPM modules and their basic configuration.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,57 @@
+{
+  "name": "@onchain-id/oid-token",
```
Comment (mplelis) [inline package.json:2]: `oid-token` should be enough no?

Comment (eloiGarrido) [reply package.json:2]: Was checking the rest of projects from `onchain-id` and they all include the mother repository. Think it similarly to openzeppelin, that they all branch from that general one

**Comment by mplelis** [comment]
@Joachim-Lebrun I am not able to approve, maybe I need more rights or something?

**Comment by eloiGarrido** [comment]
> @Joachim-Lebrun I am not able to approve, maybe I need more rights or something?

Sorry, that was me, I've whitelisted a set of people that can approve but GH didn't allow me to add you Mihalis, will try again. 😅

**Pull request merged**